runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -2470,6 +2471,10 @@ J9::ValuePropagation::transformFlattenedArrayElementLoadStoreUseTypeHint(TR_Opaq
    // "elseTree" is the fast path that uses sym ref to access array elements through "newvalue"
    TR::TreeTop *elseTree = TR::TreeTop::create(comp(), callTree->getNode()->duplicateTree());
 
+   if (trace())
+      traceMsg(comp(), "%s: ifTree n%dn (%p). slow path helper call treetop n%dn (%p). fast path newvalue treetop n%dn (%p)\n", __FUNCTION__,
+         ifTree->getNode()->getGlobalIndex(), ifTree, thenTree->getNode()->getGlobalIndex(), thenTree, elseTree->getNode()->getGlobalIndex(), elseTree);
```

Comment:
Did you really want to print the pointers to the `TR::TreeTop`s here, or did you want to print the pointers to the nodes they contain?  That is, `{if|then|else}Tree` versus `{if|then|else}Tree->getNode()`